.rubocop.yml

```diff
@@ -19,7 +21,10 @@ Layout/LineLength:
   Enabled: false
 
 Naming/FileName:
-  Exclude: ["devise-security.gemspec"]
+  Exclude:
+    - "devise-security.gemspec"
+    - 'lib/devise-security.rb'
```

Comment:
@olbrich @tmr08c @natebird what do you think about renaming these files in a separate PR?